frappe/oauth.py

```diff
@@ -252,7 +252,7 @@ def validate_bearer_token(self, token, scopes, request):
 		)
 		are_scopes_valid = True
 		for scp in scopes:
-			are_scopes_valid = are_scopes_valid and True if scp in client_scopes else False
+			are_scopes_valid = are_scopes_valid if scp in client_scopes else False
```

Comment:
```suggestion
			are_scopes_valid &= scp in client_scopes
```

Reply:
Hmm, should be able to drop the loop completely, and simplify to
`are_scopes_valid = all(scp in client_scopes for scp in scopes)`?

Reply:
yep

Reply:
cc58fd20ca2dd52a8c5761358137ee645d1716d1